Conditionally try to boot from SD card if one is present. Otherwise, boot from eMMC.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Jon Grisky <jon@resin.io>

Index: git/include/configs/mx6qvab820_common.h
===================================================================
--- git.orig/include/configs/mx6qvab820_common.h
+++ git/include/configs/mx6qvab820_common.h
@@ -216,7 +216,7 @@
 	"bootcmd_mmc=setenv mmcdev 1; setenv rootdev 0; setenv mmcname mmc; run bootcmd_main;\0"
 
 #define CONFIG_BOOTCOMMAND \
-	"run bootcmd_sd"
+	"mmc dev 0; if mmc rescan; then run bootcmd_sd; else run bootcmd_mmc; fi"
 
 #define CONFIG_ARP_TIMEOUT     200UL
 
